### ISSUE TITLE
Not Updating

### ISSUE THREAD

**Issue by strickersuk** [DasLetzteEinhorn/AlphaESS_Monitor_Hass#14]
No longer updating after the latest AlphaESS API change.

**Comment by DasLetzteEinhorn** [comment]
Yes, I noticed that too, but I haven't found a way to get new data yet.

**Comment by strickersuk** [comment]
Looks like Node Red found a way around, don't know if it helps?

https://github.com/dehsgr/node-red-contrib-alphaess

AlphaESS are really making it difficult aren't they?

**Comment by DasLetzteEinhorn** [comment]
maybe you try this integration if it fits your needs as well: https://github.com/CharlesGillanders/homeassistant-alphaESS

**Comment by strickersuk** [comment]
Thanks, I have that installed but it doesn't update regularly enough. Thanks anyway.

**Comment by bmannz** [comment]
> Thanks, I have that installed but it doesn't update regularly enough. Thanks anyway.

It's annoying that AlphaESS keep doing this to us (their customers).
I use both and Charles's one updates every minute so it seems to be the same as this one as far as I can tell.  It used to be set to 5 minutes but that could be changed and it's now set to 1min as default.

**Comment by strickersuk** [comment]
Looks like I missed that, I'll have a tweak of my dashboard then.
